sycl/include/CL/sycl/group.hpp

```diff
@@ -104,35 +104,110 @@ template <int Dimensions = 1> class group {
 
   group() = delete;
 
+  __SYCL_DEPRECATED("use sycl::group::get_group_id() instead")
   id<Dimensions> get_id() const { return index; }
 
+  __SYCL_DEPRECATED("use sycl::group::get_group_id() instead")
```

Comment:
We are using __SYCL2020_DEPRECATED macro for API deprecated in SYCL2020. Isn't this the case?